upb/descriptor/reader.c

```diff
@@ -203,6 +209,28 @@ static size_t file_onpackage(void *closure, const void *hd, const char *buf,
   return n;
 }
 
+static size_t file_onsyntax(void *closure, const void *hd, const char *buf,
+                          size_t n, const upb_bufhandle *handle) {
+  upb_descreader *r = closure;
+  UPB_UNUSED(hd);
+  UPB_UNUSED(handle);
+  /* XXX: see comment at the top of the file. */
+  if (n == strlen("proto3") && memcmp(buf, "proto3", strlen("proto3")) == 0) {
+    uint32_t i;
+    /* Cover messages created previously. */
```

Comment:
Cover doesn't seem like the best verb. How about
Update messages created before the syntax was read.

Update comment below too